engo_js.go

```diff
@@ -200,6 +207,7 @@ func rafPolyfill() {
 // RunIteration runs one iteration per frame
 func RunIteration() {
 	Time.Tick()
+	Input.clear()
 	Input.update()
 	currentWorld.Update(Time.Delta())
```

Comment:
Unrelated: I think `render` shouldn't know about `currentWorld`. This could instead be a mailbox event.